src/dataset-builder/CohortEditor.ts

```diff
@@ -592,48 +600,146 @@ export const CohortEditor: React.FC<CohortEditorProps> = (props) => {
     getNextCriteriaIndex,
   } = props;
   const [cohort, setCohort] = useState<Cohort>(originalCohort);
+  const [snapshotRequestParticipantCount, setSnapshotRequestParticipantCount] =
+    useLoadedData<SnapshotBuilderCountResponse>();
+  const defaultCohortDemographicSeries = [
+    { name: 'Asian', data: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] },
+    { name: 'Black', data: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] },
+    { name: 'White', data: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] },
+    { name: 'Native American', data: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] },
+    { name: 'Pacific Islander', data: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] },
+  ];
+  const defaultCohortAgeSeries = [{ data: [0, 0, 0] }];
+  const [cohortAges, setCohortAges] = useState<CohortDemographics>(generateCohortAgeData(defaultCohortAgeSeries));
+  const [cohortDemographics, setCohortDemographics] = useState<CohortDemographics>(
+    generateCohortDemographicData(defaultCohortDemographicSeries)
+  );
+  const countStatus = snapshotRequestParticipantCount.status;
+  function chartOptions(cohortDemographics: CohortDemographics) {
```

Comment:
The charts are meaty enough they may be worth an intermediate component, so that we can isolate the chart configuration. What are your thoughts on that?

Reply:
yeah I think that's a good idea